File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/ItemCompanyDTO.java
package com.example.befindingjob.dto;

import com.example.befindingjob.entity.Company;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.checkerframework.checker.units.qual.N;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ItemCompanyDTO {
    private Integer companyId;
    private String companyName;
    private String logoUrl;
    private String description;

    public ItemCompanyDTO(Company company) {
        this.companyId = company.getCompanyId();
        this.companyName = company.getCompanyName();
        this.logoUrl = company.getLogoUrl();
        this.description = company.getDescription();
    }
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/ItemCompanyDTO.kt
package com.example.jobhub.dto

import com.google.gson.annotations.SerializedName

data class ItemCompanyDTO(
    @SerializedName("companyId") val companyId: Int,
    @SerializedName("companyName") var companyName: String? = "",
    @SerializedName("logoUrl") val logoUrl: String? = "",
    @SerializedName("description") var description: String? = "",
)

